nvm.sh

```diff
@@ -54,7 +54,7 @@ nvm_command_info() {
   local INFO
   COMMAND="${1}"
   if type "${COMMAND}" | command grep -q hashed; then
-    INFO="$(type "${COMMAND}" | command sed -E 's/\(|)//g' | command awk '{print $4}')"
+    INFO="$(type "${COMMAND}" | command sed -E 's/\(|\)//g' | command awk '{print $4}')"
```

Comment:
The origin one will cause error on GNU sed:
`sed: -e expression #1, char 9: Unmatched ) or \)`